File: 백준/Bronze/34032. ／gg/／gg.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
int main(void) {
    int n, mid, cnt = 0;
    char* st = (char*)malloc(sizeof(char) * 500001);
    scanf("%d %s", &n, st);
    mid = n / 2 + (n % 2 == 0 ? 0 : 1);
    for (int i = 0; i < strlen(st); i++) {
        if (st[i] == 'O')
            cnt += 1;
    }
    printf("%s", cnt >= mid ? "Yes" : "No");
    return 0;
}

File: 백준/Bronze/34032. ／gg/／gg.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
public class Main {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int cnt=0,mid,n = Integer.parseInt(br.readLine());
        String st = br.readLine();
        mid=n/2+(n%2==0?0:1);
        for(int i=0;i<st.length();i++) {
            if(st.charAt(i)=='O')
                cnt+=1;
        }
        System.out.print(cnt>=mid?"Yes":"No");
    }
}